tools/cli/helpers/docker-config.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * External dependencies
+ */
+import { readFileSync, writeFileSync, existsSync } from 'fs';
+import yaml from 'js-yaml';
+import path from 'path';
+import { merge } from 'lodash';
+
+export const dockerFolder = `tools/docker`;
+const overrideConfigFile = '.docker-env.json';
+
+const mergeDockerVolumeMappings = ( mainMapping, overrideMapping ) => {
+	const volumesMapping = overrideMapping.slice(); // make a copy of an array;
+
+	const typeLocPaths = overrideMapping.map( map => map.split( ':' )[ 0 ] );
+	const defaultLocPaths = mainMapping.map( map => map.split( ':' )[ 0 ] );
+	defaultLocPaths.forEach( ( defaultPath, id ) => {
+		if ( ! typeLocPaths.includes( defaultPath ) ) {
+			volumesMapping.unshift( mainMapping[ id ] );
+		}
+	} );
+
+	return volumesMapping;
+};
+
+/**
+ * DEPRECATED. Parses compose-volumes.yml and adds it's contents into docker config
```

Comment:
This function isn't really deprecated as the doc comment implies. It implements a deprecated feature.
```suggestion
 * Parses deprecated compose-volumes.yml and adds its contents into docker config.
```